content/applications/services/field_service/worksheets.rst

```diff
@@ -0,0 +1,72 @@
+===================
+Worksheet Templates
+===================
+
+**Worksheets** help your field service workers perform and report the tasks they do on site.
+They can feature various information, such as instructions, to-do lists, etc. You can
+also format your worksheet using checkboxes, bullet points, blank fields to fill in, HTML, and add
+files, images, links, and more.
+
+Because it is common for businesses to have their workers perform the same type of field service
+repeatedly, Odoo offers the option to create custom **Worksheet Templates**. By doing so, you
+don’t have to create the same worksheet every time you plan the same type of field service task.
+
+Configuration
+=============
+
+To use **worksheets** in **Field Service**, the **Worksheet Template** feature must be enabled
+first. Go to :menuselection:`Field Service --> Configuration --> Settings --> Worksheet Templates`
+
+Create a Worksheet Template
+===========================
+
+To create your **worksheet templates**, go to :menuselection:`Field Service --> Configuration -->
+Worksheet Templates`. Click :guilabel:`New`, give your worksheet template a name, then click
+:guilabel:`Design Template`. Doing so will open the **Studio**.
+
+.. important::
+   Clicking :guilabel:`Design Template` will automatically install the **Studio** app in your
+   database, which is required to design worksheet templates. Adding more apps to your database may
+   impact your price plan.
+
+In the Studio click :guilabel:`Add`. Drag and drop the items that you want from the left column
+into your worksheet. To rearrange the items, just drag and drop them in the order that you want.
+Click on the items you added to your worksheet to customize their properties.
+
+When your Worksheet Template is complete, click :guilabel:`Close` in the top right corner of the
+page to leave **Studio**.
+
+Add a Worksheet Template to a Field Service task
+================================================
+
+Go to your Field Service task and select a :guilabel:`Worksheet Template`. Remember that the
+:guilabel:`Task Title` and :guilabel:`Customer` fields are mandatory. Click :guilabel:`Save`.
+
+Assign a default Worksheet Template
+===================================
+
+If you have configured multiple Worksheet Templates, you can select one to be set up by
+default when you create a new Field Service task. Go to your task, drag your mouse over the name of
+your project to make the green arrow and the **internal link** tab appear, click on it.
```

Comment:
I don't think the title is necessary here and it looks a little odd with just 1 line udner the previous one. Instead, I would maybe do something like this to 1) introduce the concept of a default template (which was done in the title) and 2) be a little more direct:

```suggestion
By default, the :guilabel:`Default worksheet` template is selected. To define another default worksheet template,  
click the :icon:`oi-arrow-right` (right-arrow) button that appears when you hover your mouse over the :guilabel:`Project` field on the task form. 

```
see this doc for how to add icons directly into the text like I did in my suggestion https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#icons

Reply:
Thank you for this tip!